src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -142,5 +144,30 @@ export const useContentTaxonomyTagsUpdater = (contentId, taxonomyId) => {
     onSettled: () => {
       queryClient.invalidateQueries({ queryKey: ['contentTaxonomyTags', contentId] });
     },
+    onSuccess: () => {
```

Comment:
This notification is just temporary. In the future, the course outline page and course unit page will be part of the frontend-app-course-authoring, so there will be no need to send a change notification nor to refresh the counts - React Query will handle that all automatically :)

So: I want to make sure everyone who sees this code understands that. Please add a comment that says something like "In the future, when the Course Outline Page and Unit Page are integrated into this MFE, they should just use React Query to load the tag counts, and React Query will automatically refresh those counts when the mutation invalidates them. So this postMessage is just a temporary feature to support the legacy Django template courseware page."

Reply:
Updated here: https://github.com/openedx/frontend-app-course-authoring/pull/800/commits/78db929864f04038023d5f1374a9f8048791f880

Reply:
Nice, thanks!